code/controllers/subsystem/ambience.dm

```diff
@@ -21,8 +21,10 @@ SUBSYSTEM_DEF(ambience)
 
 		var/area/current_area = get_area(client_iterator.mob)
 
-		var/sound = pick(current_area.ambientsounds)
+		var/ambience = pick(current_area.ambientsounds)
+		if(!ambience)
```

Comment:
This can result in `error: pick from empty list`

Consider using `safepick`

Reply:
Ah, I'd assumed that the `pick()` proc would just pass on `null` if it was empty.